great_expectations/data_context/data_context.py

```diff
@@ -927,76 +929,108 @@ def profile_datasource(self,
             When success = False, the error details are under "error" key
         """
 
+        datasource = self.get_datasource(datasource_name)
```

Comment:
This variable isn't used anywhere

Reply:
We don't need the datasource object, but this line serves to check if the datasource by the name passed as an arg exists and raise an error if it does not. Added a comment to the code.